### PR TITLE
"Use the docker image in sample app"

### PR DESCRIPTION
**Pull request by kowshikn**
Issue #212 

Using `restart` for data prepper to start repeatedly till ODFE becomes available.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/adot/docker-compose.yml [hunk 1]
```diff
@@ -27,16 +27,12 @@ services:
...
       - my_network
   data-prepper:
+    restart: unless-stopped
     container_name: data-prepper
-    build:
-      context: ../..
-      dockerfile: examples/Dockerfile
-    working_dir: /app
-    command: sh data-prepper-wait-for-odfe-and-start.sh
+    image: opendistroforelasticsearch/data-prepper:v0.7.0-alpha
```
Comment (yadavcbala) [inline examples/adot/docker-compose.yml:32]: this is staging repository - you can as well use production after Dec14

**Review by yadavcbala — APPROVED** [review]

**Comment by yadavcbala** [comment]
Also, since you are using the image for data-prepper, the examples will not be able to use any non-released features - I am assuming you want it to be like this

File: examples/adot/docker-compose.yml (hunk 1 above)
Comment (kowshikn) [reply examples/adot/docker-compose.yml:32]: Okay. Will follow up with another PR on 14th.

**Review by chenqi0805 — APPROVED** [review]
LGTM. Same ques as @yadavcbala . I am OK with the change. But do we want customer to run examples on releases or on the bleeding edge of the data-prepper.

**Comment by kowshikn** [comment]
> LGTM. Same ques as @yadavcbala . I am OK with the change. But do we want customer to run examples on releases or on the bleeding edge of the data-prepper.

Releases, because we want them to pull the latest approved version and also avoid gradle build etc.

**Pull request merged**